cli/lib/tasks/download.js

```diff
@@ -196,32 +196,27 @@ const downloadFromUrl = ({ url, downloadDestination, progress, ca }) => {
       downloadDestination,
     })
 
-    let redirectVersion
+    if (ca) debug('using custom CA details from npm config')
```

Comment:
nit: i know there isn't a eslint rule for this, but i think this reads better.

```suggestion
    if (ca) {
      debug('using custom CA details from npm config')
    }
```

Reply:
well, alright.  9_9